manifests/resource/mailhost.pp

```diff
@@ -102,10 +153,10 @@
   Optional[String] $ssl_trusted_cert             = undef,
   Optional[Integer] $ssl_verify_depth            = undef,
   Enum['on', 'off', 'only'] $starttls            = 'off',
-  $protocol                                      = undef,
+  Optional[Enum['imap', 'pop3', 'smtp']] $protocol = undef,
   Optional[String] $auth_http                    = undef,
   Optional[String] $auth_http_header             = undef,
-  String $xclient                                = 'on',
+  Enum['on', 'off'] $xclient                     = 'on',
```

Comment:
Same here: previously documented and docs agree: https://nginx.org/en/docs/mail/ngx_mail_proxy_module.html#xclient